app/services/notifications/new_mention/send.rb

```diff
@@ -0,0 +1,36 @@
+module Notifications
+  module NewMention
+    class Send
+      delegate :user_data, to: Notifications
+      delegate :comment_data, to: Notifications
+
+      def initialize(mention)
+        @mention = mention
+      end
+
+      def self.call(*args)
+        new(*args).call
+      end
+
+      def call
+        Notification.create(
```

Comment:
I don't recall but, can this ever fail?

Reply:
Like everything else :thinking: Probably, `create!` would be better?

Reply:
I guess it depends if you want to be notified of the error or not :D

Reply:
Updated that. Noticed several other similar places where I should do this.